generator/internal/sample/serviceconfig.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Google LLC
```

Comment:
nit: if this is a new file then it should be 2025.  My guess is you moved it but GitHub gets confused when you move and change a file "too much".

Reply:
Done.